app.py

```
@@ -97,8 +95,8 @@ def save_chats_to_local():
 
         # Save usage data
         st.session_state.local_storage.setItem("mbu_usage", json.dumps(st.session_state.usage_data, default=str))
-    except Exception as e:
-        print(f"Error saving to localStorage: {e}")
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)